package.json

```diff
@@ -10,7 +10,8 @@
     "lerna:publish": "yarn install && yarn run lerna:prepublish && lerna publish from-package --ignore-scripts --ignore-prepublish",
     "test": "TZ='UTC' jest",
     "test:watch": "TZ='UTC' jest --watch",
-    "lint": "eslint {packages,app}/**/*.{js,ts,tsx} --max-warnings 0"
+    "lint": "eslint {packages,app}/**/*.{js,ts,tsx} --max-warnings 0",
+    "postinstall": "patch-package"
```

Comment:
yeap, it adds a patch to one of the node packages

Reply:
shouldn't it be used in the start script then? like `yarn run postinstall && yarn run start`

Reply:
the patch package script should ideally run after every yarn install, instead of doing so manually the postinstall script does so automatically after each `yarn install` or  `yarn add`